README.md

```diff
@@ -59,6 +59,10 @@ If `true` it will send HTTP POST requests to a given url.
 
 The url where to send HTTP POST requests (default: `http://localhost`).
 
+`weblog.basicToken`
+
+A Basic authentication token to append to each HTTP request.
```

Comment:
This is too generic. If I'm understanding correctly the user should specify the password as a base64 encoded string. This should be explained in the readme.

Reply:
done